### PR TITLE
feat: add assistant from new scratch for Python template

### PR DESCRIPTION
**Pull request by frankqianms**
workitem: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/27551355

create the app:
image
image
image
image

run the app:
image

### CODE REVIEW

File: packages/fx-core/src/question/create.ts [hunk 1]
```diff
@@ -1452,7 +1452,9 @@ export function getLanguageOptions(inputs: Inputs): OptionItem[] {
     // SPFx only supports typescript
     return [{ id: ProgrammingLanguage.TS, label: "TypeScript" }];
   } else if (
-    capabilitiesHavePythonOption.includes(capabilities) &&
+    capabilitiesHavePythonOption.includes(
+      inputs[capabilities] ? inputs[capabilities] : capabilities
```
Comment (blackchoey) [inline packages/fx-core/src/question/create.ts:1456]: why we need to add check for `input[capabilities]` here?

File: templates/python/custom-copilot-assistant-new/teamsapp.local.yml.tpl [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.3/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.3
```
Comment (blackchoey) [inline templates/python/custom-copilot-assistant-new/teamsapp.local.yml.tpl:4]: I remember Huajie changes the version to v1.5. Please also update the version here. Can sync with him first.

File: packages/fx-core/src/question/create.ts (hunk 1 above)
Comment (frankqianms) [reply packages/fx-core/src/question/create.ts:1456]: for custom-copilot-rag-customize and custom-copilot-rag-azureAISearch, there capabilities are both `custom-copilot-rag` but `inputs[custom-copilot-rag]` are different.
I want to make the judgement more specific on app type not capability type.

File: templates/python/custom-copilot-assistant-new/teamsapp.local.yml.tpl (hunk 2 above)
Comment (frankqianms) [reply templates/python/custom-copilot-assistant-new/teamsapp.local.yml.tpl:4]: sure

**Review by blackchoey — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**